src/renderer/components/+network-services/service-port-component.tsx

```diff
@@ -61,14 +104,32 @@ export class ServicePortComponent extends React.Component<Props> {
   render() {
     const { port } = this.props;
 
+    if (this.forwardPort == -1) {
+      this.forwardPort = port.port;
+    }
+
+    const portForwardAction = async () => {
+      if (this.isPortForwarded) {
+        await this.stopPortForward();
+      }else {
+        await this.portForward();
+      }
+    };
+
     return (
       <div className={cssNames("ServicePortComponent", { waiting: this.waiting })}>
-        <span title="Open in a browser" onClick={() => this.portForward() }>
-          {port.toString()}
-          {this.waiting && (
-            <Spinner />
-          )}
-        </span>
+        {port.toString()}
+        {" "}
+        <text>to</text>
+        <Input className={"portInput"}
+          type="number"
+          min="0"
+          max="65535"
+          value= {String(this.forwardPort)}
+          disabled={this.isPortForwarded}
+          onChange={(value) => this.forwardPort = Number(value)}
+        />
+        <Button onClick={() => portForwardAction()}> {this.isPortForwarded ? "Stop":"Forward"} </Button>
```

Comment:
Please add some screenshots of how this looks

Reply:
![2021-07-09_23-37](https://user-images.githubusercontent.com/12953177/125150574-c1136e80-e10e-11eb-99bf-99cdde11276b.png)


![2021-07-09_23-39](https://user-images.githubusercontent.com/12953177/125150597-fd46cf00-e10e-11eb-9eec-57f07bdde740.png)

Reply:
Thanks for the feedback, I have made the changes as suggested